app/main.py

```diff
@@ -1,3 +1,66 @@
+from typing import Any
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Any) -> object:
```

Comment:
The return type annotation for `__add__` should be `Distance` instead of `object` to reflect the actual return type.